include/unix/osd.h

```diff
@@ -129,7 +129,7 @@ static inline ssize_t ofi_recv_socket(SOCKET fd, void *buf, size_t count,
 static inline ssize_t ofi_recvfrom_socket(SOCKET fd, const void *buf, size_t count, int flags,
 					  const struct sockaddr *to, socklen_t tolen)
 {
-	return sendto(fd, buf, count, flags, to, tolen);
+	return recvfrom(fd, buf, count, flags, to, tolen);
```

Comment:
hmm :)
Thanks for catching this

Reply:
Sean found it when we were checking the WSA calls for windows.